tests/integration/test_integration.py

```diff
@@ -299,6 +299,11 @@ def test(
         # Verify that command completed successfully
         assert exit_code == Status.OK
 
+        expected["metadata"]["component"]["bom-ref"] = actual["metadata"]["component"][
+            "bom-ref"
+        ]
+        expected["dependencies"][0]["ref"] = actual["metadata"]["component"]["bom-ref"]
+
```

Comment:
As it's odd to set expected = actual and then later test for `expected == actual`, I would kindly ask you to remove this and replace it with removing the ref completely from the `expected` and `actual` and test just additionally that `actual["metadata"]["component"]["bom-ref]"` is a valid uuid, i.e. utilizing `uuid.UUID(dict)`

Reply:
Changed